src/Transport/DefaultTransportFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Transport;
+
+use Http\Message\MessageFactory as MessageFactoryInterface;
+use Sentry\HttpClient\HttpClientFactoryInterface;
+use Sentry\Options;
+
+final class DefaultTransportFactory implements TransportFactoryInterface
```

Comment:
Please add the docblock with the description of what this class does, e.g. `This class is the default implementation of the {@see TransportFactoryInterface} interface.`